pom.xml

```diff
@@ -155,89 +156,21 @@
           </executions>
         </plugin>
         <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-antrun-plugin</artifactId>
-          <version>1.7</version>
+          <groupId>co.cask</groupId>
+          <artifactId>cdap-maven-plugin</artifactId>
+          <version>1.0.0-SNAPSHOT</version>
+          <configuration>
+            <cdapArtifacts>
+              <parent>system:cdap-data-pipeline[4.2.0,9.0.0-SNAPSHOT)</parent>
```

Comment:
this changed, right? Intentional?

Reply:
Yea. I figured its no point disallowing Hive plugins against 5.0, since it is compatible.